### PR TITLE
smoke_test: change generator to produce <=10 sim'able tasks

### PR DESCRIPTION
**Pull request by dbentley**
Change sim to take a sleep task

### CODE REVIEW

File: scootapi/testhelpers/generators.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func GenJobDefinition(rng *rand.Rand) *scoot.JobDefinition {
 func GenTask(rng *rand.Rand) *scoot.TaskDefinition {
 
 	cmd := scoot.NewCommand()
-	cmd.Argv = []string{"sleep", "1"}
+	cmd.Argv = []string{"sleep 500", "complete 0"}
```
Comment (jasonbs10) [inline scootapi/testhelpers/generators.go]: I think we need to be more opinionated and/or more configurable with regard to testing. Do you want to make the default runner defer to simExec? Or complete the TODO of adding configuration for runner and worker, and specify those in our smoketests?

Comment (jasonbs10) [inline scootapi/testhelpers/generators.go]: Right, but this PR could do more to be end-to-end. Unless I'm misreading, we use os/execer and by redefining tasks here, you'll break anything that currently runs scootapi.smoketest.

Comment (dbentley) [inline scootapi/testhelpers/generators.go]: OK, how's that?

Comment (dbentley) [inline scootapi/testhelpers/generators.go]: (To clarify: I changed workerserver/main.go to now use a SimExecer)

Comment (jasonbs10) [inline scootapi/testhelpers/generators.go]: That should be ok.

**Comment by jasonbs10** [comment]
lgtm

**Pull request merged**